profile-test/tests/profile/analyze.rkt

```diff
@@ -107,3 +107,16 @@
      [bad (error 'test ">>> ~s" bad)])
 
    ))
+
+(provide merge-test)
+(module+ main (merge-tests))
```

Comment:
Is there a reason for this to be in `main` rather than `test`?

Reply:
No clue but the other things in this file also use `main` instead of `test`

Reply:
Ah, ok.